src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>Microsoft.ApplicationInsights.AspNetCore</AssemblyName>
-    <VersionPrefix>2.7.0-beta3</VersionPrefix>
+    <VersionPrefix>2.8.0-beta24</VersionPrefix>
```

Comment:
expected?

Reply:
Yep, testing for a while, will map to Base SDK with Perf changes once it's merged.